src/System.IO.FileSystem/tests/packages.config

```diff
@@ -3,6 +3,7 @@
   <package id="System.Diagnostics.Debug" version="4.0.10-beta-22703" targetFramework="portable-net45+win+wpa81" />
   <package id="System.IO" version="4.0.10-beta-22703" targetFramework="portable-net45+win+wpa81" />
   <package id="System.IO.FileSystem.Primitives" version="4.0.0-beta-22703" targetFramework="portable-net45+win+wpa81" />
+  <package id="System.IO.Pipes" version="4.0.0-beta-22703" targetFramework="portable-net45+win+wpa81" />
```

Comment:
Thanks; good to know.  If you don't mind, let's defer this to when we actually need to run the tests some place they're not available.